index.bs

```diff
@@ -1023,36 +1023,106 @@ Issue: Define how this works.
 
 ## Reference ## {#data-types-reference}
 
-<dfn>RemoteReference</dfn> represents a remote reference to an exising ECMAScript object in
-[=object id map=].
-
-A [=BiDi session=] has an <dfn>object id map</dfn>. This is a weak map
-from objects to their corresponding id.
-
-Issue: Should this be explicitly per realm?
+<dfn>RemoteReference</dfn> represents a remote reference to an existing
+ECMAScript object ([=RemoteObjectReference=]), or a {{SandboxProxy}}
+([=SharedReference=]).
+
+<dfn>RemoteObjectReference</dfn> represents a remote reference to an exising
+ECMAScript object in [=object id map=].
+
+<dfn>SharedReference</dfn> represents a remote reference to an existing or
+created on-demand {{SandboxProxy}} of an object in [=shared id map=].
+
+In [=BiDi session=]:
+ * Each realm has an <dfn>object id map</dfn>, a weak map from objects to their
+   corresponding id. The [=object id map=] is unique per realm.
+ * Each realm has a <dfn>shared id to proxy map</dfn>, a weak map
+   from shared id to their corresponding {{SandboxProxy}} in
+   the given realm. The [=shared id to proxy map=] is unique per
+   realm.
+ * Each browsing context has a <dfn>shared id map</dfn>, a weak map from
+   ECMAScript object to their corresponding shared id. The
+   [=shared id map=] is shared among all of the realms of the context's
```

Comment:
I assume this shouldn't be shared between all the realms of the `context` but the `Window` so that it matches what we want to land via https://github.com/w3c/webdriver/pull/1667?